pages/game_page.py

```diff
@@ -1,3 +1,33 @@
 import streamlit as st
+import random
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Clinical Trial','Mutation','Genome','Oncology']
+
+def game_word_generator(word): 
+    st.write(word)
+    st.write(f"`{' '.join(['_'] * len(word))}`")
+    
+if game_words:
+    selected_word = random.choice(game_words)
+    game_word_generator(selected_word)
+   
+# Session state
+if 'selected_word' not in st.session_state:
+    st.session_state.selected_word = game_word_generator(selected_word)
+
+# Create a text box for the player to enter a letter
+guess = st.text_input("Guess a letter:")
```

Comment:
If you add the "key" parameter as shown [here](https://github.com/amy-morrill/coding-club-group-8/blob/main/test_script_streamlit.py#L55) it will save the input in session state automatically. So for example if this is changed to 
```
st.text_input("Guess a letter:", key="guess")
```
Then you will be able to access the guess using:
```
st.session_state.guess
```